cassandra.go

```diff
@@ -91,8 +105,15 @@ func (c *cassandra) Close() error {
 	return nil
 }
 
+// GetQuery provides an access to the gocql.Query if a user of this library needs to tune some parameters for
+// a specific query without modifying the parameters the library was configured with, for example to use
+// a consistency level that differs from the configured read/write consistency levels.
+func (c *cassandra) GetQuery(consistency gocql.Consistency, queryString string, queryParams ...interface{}) *gocql.Query {
```

Comment:
Minor issue, since this does the same thing as `session.Query` is there a reason it's named differently?